### PR TITLE
docs: update computed-watch-api.md

### PR DESCRIPTION
**Pull request by wxsms**
## Description of Problem

#626

### CODE REVIEW

File: src/api/computed-watch-api.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ interface DebuggerEvent {
 
 ## `watchEffect`
 
-在响应式地跟踪其依赖项时立即运行一个函数，并在更改依赖项时重新运行它。
+在响应式地跟踪其依赖项时立即运行一个函数，并在依赖项发生变化时重新运行它。
```
Comment (zhanglolo) [inline src/api/computed-watch-api.md:64]: 英文文档中
api/computed-watch-api.html#watcheffect原文：
> Runs a function immediately while reactively tracking its dependencies and re-runs it whenever the dependencies are changed.

guide/reactivity-computed-watchers.html#watcheffect原文：
> To apply and automatically re-apply a side effect based on reactive state, we can use the watchEffect method.**It runs a function immediately while reactively tracking its dependencies and re-runs it whenever the dependencies are changed**.

第一段和第二段粗体部分是一样的。

所以这里建议使用和中文文档中reactivity-computed-watchers.html#watcheffect这里的翻译：

立即执行传入的一个函数，同时响应式追踪其依赖，并在其依赖变更时重新运行该函数。

**Review by zhanglolo — COMMENTED** [review]
提了一点点修改意见

File: src/api/computed-watch-api.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ console.log(plusOne.value) // 2
 plusOne.value++ // 错误
 ```
 
-或者，它也可以使用具有 `get` 和 `set` 函数的对象来创建可写的 ref 对象。
+或者，它也可以是具有 `get` 和 `set` 函数的对象，以创建可写的 ref 对象。
```
[the remark this answers is not in the document]
Comment (wxsms) [reply src/api/computed-watch-api.md:18]: 确实。

```suggestion
或者，接受一个具有 `get` 和 `set` 函数的对象，用来创建可写的 ref 对象。
```

File: src/api/computed-watch-api.md (hunk 1 above)
Comment (wxsms) [reply src/api/computed-watch-api.md:64]: 你读得很仔细呀 👍 

```suggestion
立即执行传入的一个函数，同时响应式追踪其依赖，并在其依赖变更时重新运行该函数。
```

**Review by veaba — APPROVED** [review]

**Pull request merged**